includes/Admin/Editor.php

```diff
@@ -215,7 +215,7 @@ public static function get_requirements(): array {
 	 * @param WP_Post    $post    Current post object.
 	 * @return bool|mixed Whether the editor has been replaced.
 	 */
-	public function replace_editor( $replace, $post ) {
+	public function replace_editor( $replace, $post ) { // phpcs:ignore SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint
```

Comment:
Weird, did you get PHPCS errors otherwise? Why now and why not on `main` as well? 🤔